docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -525,6 +531,90 @@ The available history information is an array of upgrade information - upgrade s
 noobaa-cli upgrade history
 ```
 
+## Managing Connections
+
+- **[Add Connection](#add-connection)**: Create new connections with customizable options.
+- **[Update Connection](#update-connection)**: Modify the settings and configurations of existing connections.
+- **[Connection Status](#connection-status)**: Retrieve the current status and detailed information about a specific connection.
+- **[List Connections](#list-connections)**: Display a list of all existing connections.
+- **[Delete Connection](#delete-connection)**: Remove unwanted or obsolete connections from the system.
+
+### Add Connection
+
+The `connection add` command is used to create a new connection with customizable options.
+
+#### Usage
+```sh
+noobaa-cli connection add --from_file
```

Comment:
From what I understand the `from_file` option was used by the team from ODF who tested with it, but was not used by Spectrum. Do you plan to create an alternative command for the connection add?

Reply:
You can use regular ADD, but it's not convenient as some value are of type object, which cannot be parsed from command line.